tests/scheduling_utils.py

```diff
@@ -212,7 +221,12 @@ def check_scheduler_inference_steps(
                 new_token_ids[0])
             collected_outputs[output.request_id]["logprobs"].append(
                 new_logprobs[0][0])
+            collected_outputs[output.request_id]["tokens"].append(
```

Comment:
I'm not sure I understand why the decoding is needed. Is it just to print text for debugging instead of token indices?